spinta/manifests/tabular/helpers.py

```diff
@@ -1969,19 +1974,36 @@ def load_ascii_tabular_manifest(
 
 
 def get_relative_model_name(dataset: [str, dict], name: str) -> str:
+    # First handle any feature parts
+    base_name = name
```

Comment:
I'd suggest to name this `basename` as one word, because it's like that in other places and `base_name` somehow suggests that it's the name of the base, i.e. base as the dimension of DSA